enterprise/server/execution_service/execution_service.go

```diff
@@ -17,6 +17,8 @@ import (
 	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
 	timestamppb "github.com/golang/protobuf/ptypes/timestamp"
 	statuspb "google.golang.org/genproto/googleapis/rpc/status"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/protobuf/proto"
```

Comment:
nit: i assume vim did this, but could you please put the "unaliased" imports up above in a separate section from the aliased ones?

Reply:
Done